### DISCUSSION TITLE
Antivirus error.

### DISCUSSION THREAD

**Discussion by JohntyMacc** [DeckCheatz/wemod-launcher#112] [Q&A] [answered]
Hi there

Any idea how to go about fixing this? Seems to have become a problem in the last few weeks after everything previously working fine.

I’ve reinstalled, deleted prefixes and so on. No luck.

![IMG_5790](https://github.com/user-attachments/assets/3e2c1be2-2242-4221-b9ce-74cdc622b99a)

**Comment by marvin1099** [comment]
The prefix install has failed when this message appears.
Send over the log file, I will take a look at it.

**Comment by JohntyMacc** [comment, marked as answer]
Hi there

I think I figured it out.

I’d upgraded to a bigger micro SD and reinstalled a lot of my games to it
without uninstalling them from the old one.

It caused a few other issues as well so in the end I did a factory reset of
my deck and reinstalled the lot. It’s all working fine now but
unfortunately for obvious reasons I do t have the log file to confirm.

Just want to say thank you for the work you do on this, it’s truly an
excellent addition and much appreciated.

Many Thanks
Johnstone

**Comment by marvin1099** [comment]
Perfect
